README.md

```diff
@@ -147,6 +147,8 @@ acquire this dependency.
     * We recommend selecting "C++ CMake tools for Windows" in the VS Installer.
     This will ensure that you're using supported versions of CMake and Ninja.
     * Otherwise, install [CMake][] 3.20 or later, and [Ninja][] 1.10.2 or later.
+    * We recommend selecting "Python 3".

```

Comment:
The VS Installer offers "Python 3 32-bit" and "Python 3 64-bit". Presumably we should recommend "Python 3 64-bit".